SRC/cgetv0.f

```diff
@@ -271,9 +269,9 @@ subroutine cgetv0
 c
       call arscnd (t2)
       first = .TRUE.
+      call ccopy (n, workd(n+1), 1, resid, 1)
```

Comment:
Why is this needed? We only end up here if ido != 0.

Reply:
I'm away from my laptop, but aftet the very first OP application ido is -1, isn't it? So we need to copy the result of OP application back to resid.